client/my-sites/plans/jetpack-plans/style.scss

```
@@ -4,78 +4,24 @@
 /* cyrillic-ext */
 @font-face {
 	font-family: Inter;
-	font-style: normal;
-	font-weight: 400;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Regular.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Regular.woff?v=3.19) format("woff");
-}
-@font-face {
-	font-family: Inter;
-	font-style: italic;
-	font-weight: 400;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Italic.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Italic.woff?v=3.19) format("woff");
-}
-
-@font-face {
-	font-family: Inter;
-	font-style: normal;
-	font-weight: 500;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Medium.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Medium.woff?v=3.19) format("woff");
-}
-@font-face {
-	font-family: Inter;
-	font-style: italic;
-	font-weight: 500;
+	font-weight: 100 900;
```

In order for the TeamCity "Code style (Web app)" check to pass, you may need to disable the Stylelint(scales/font-weights) rule for this line:

```suggestion
	/* stylelint-disable-next-line scales/font-weights */
	font-weight: 100 900;
```

---

Thanks, @elliottprogrammer — I've updated both instances where this exists.